plugin/src/software/aws/toolkits/eclipse/amazonq/lsp/AmazonQLspClientImpl.java

```diff
@@ -170,4 +178,92 @@ public final void ssoTokenChanged(final SsoTokenChangedParams params) {
             Activator.getLogger().error("Error processing " + kind + " ssoTokenChanged notification", ex);
         }
     }
+    public enum NotificationSeverity {
+        LOW, MEDIUM, HIGH
+    }
+
+    /**
+     * Shows a notification to the user with the specified content and severity.
+     * This method is designed to be overridden by subclasses that need to customize
+     * the notification behavior.
+     *
+     * @param params The notification parameters containing the message type and content
+     */
+    @Override
+    public final void showNotification(final NotificationParams params) {
+        Activator.getLogger().info("Received notification JSON: " + params.type().toString());
```

Comment:
In general we will want to avoid logging these messages verbatim. Or is this the type? Then probably `Received notification of type <TYPE>`.